doc/howto/migrate_from_lxc.md

```diff
@@ -0,0 +1,94 @@
+(migrate-from-lxc)=
+# How to migrate containers from LXC to LXD
+
+LXD provides a tool (`lxc-to-lxd`) that you can use to import LXC containers into your LXD server.
+The LXC containers must exist on the same machine as the LXD server.
+
+The tool analyzes the LXC containers and migrates both their data and their configuration into new LXD containers.
+
+```{note}
+Alternatively, you can use the `lxd-migrate` tool within a LXC container to migrate it to LXD (see {ref}`import-machines-to-instances`).
+However, this tool does not migrate any of the LXC container configuration.
+```
+
+## Get the tool
+
+If you're using the snap, the `lxc-to-lxd` is automatically installed.
+It is available as `lxd.lxc-to-lxd`.
+
+Otherwise, make sure that you have `go` installed and get the tool with the following command:
+
+    go get github.com/lxc/lxd/lxc-to-lxd
```

Comment:
Is this still the correct way to install it?
My test failed because of the go version, but apart from that it seemed to do what it was supposed to.

Reply:
I wonder if we should build assets like we do for lxd-migrate (https://github.com/lxc/lxd/releases/tag/lxd-5.5)

@stgraber ?

Reply:
We can't because it can't be static.